node/res/bp_parachain.json

```diff
@@ -0,0 +1,48 @@
1eecf4c7f3b9b32da80ad78ef580bd40768bfd430ce4a9944229273c8a8af25237fa3d267e5a12f0388f678c04c6bbbf085defe005cfd801ac0b13e6fc57a1e44feb0d78a2fd07671ea04f817e8ecdf2f54ee39083eb2ed7f3e1371530b23d1a1365a17e535bc30db3d340709e5ef04dbc971fa97123390bfd609a01be057c73c774c1c2753b5fbdc66a481ea52c04bc6fcf8c0f705b09eb46b4b97057956b405a41c4a8f3e0d32caef0a66f685a956e78b184e967e2d310d9221ec27bdeb512fd2e300c46d5188626a0be3882d009613d0ecb546cc1ac93f759b07ff16baf14dbec99f794837d88dc56543f0f0ad20aeb3d4c0cb9b75f06536da4e0a20f704a3a7010e023a50151085f34c7eb50a7e7db5e889c50844fb38eb848806acaf993af5be5b46c7e27cfabbe1ad73a100a303a6f869af2dad3fd5efbc1a2cf4ce5c66436b323622dea54c2e11a5d26522c827c775912968943c854c92c67457cd9bca3a8032ee3607e6c3e603a1f713c902bfb681e9f00d9bb362a686b3b37ae6cc7560ca2d8181846b6332f426d5bcd3844059ae187749db4b8f7c107671dead26739d7738293c71106cf27aea260e92a8fb206ed66292695e6bc56a9cedb6786b716d43bc3870fc38e4049355497d9dad3cd0df6feebeba72011737805d805c24c2a41556494ee6401fcfc4b2517067a0ac91d8079507a78c57ff046a403b7719670fc467edbb61f767136fe3eb96e6fb07ac0047ca0013301239283fee7ce092517136157240376c2fd7980c6993c9b48f4d70f5d3c0b95696bc3947a7d1ee734351b36f44b7a64e10aff8e11bf37a6f5a2eeb189a71621826dba476955e1d949cbc9b10baec665dbb3ce887bff2a6bf8080beb3bab474b8904420c6ef90b17b4b15957966bf8c76e496cc53a3b17779d38b5c5b0ca5f8962464c8f8241782797f315fba9da8c3d1da7b0ebbf8bb751500695e3cc31bc56b0622a4c759073ab4c9b03e4a4bc0b554ec5fb3b2c7b656fe7a58a8152c0674074b661bba8a4a15b3f4425fed2e857ca150170633f45fd397db5076edd79678f4aec16e45530cb526c12a97a850584a25501b416b5d9f0ef5a004381f562f6f2cf03d155229ae6c15998f53583e246338a31e192e0ca01845bc8859142b610c8b5a4c400be9f5c2c97cede8e2c7ad3f02a50a78eb5840d6a9d8bd7261adb112b5b3f678916e3496e1f84b77922f0d0713245d8424214bdcf952232f9f3302642c4d5b054bee457c561584d9ea68cb22fab6227757905c45ec441c5c16fc8b64d6f160dc2c8c3445a4147c12b13f21913ca765709dab2c58b5567a8931b6b38d0da2164c776431ec29ab30d5db1513135c82429dcac0d89f55f2a7100fd4b2565beb02eb2a506a78d2196656e54c6d85d19e67430bf74749b3bac76c31fb9b45d9c3e34e203ef3d44db952eb451a786b9ebcc1ca9c3273a6e5ef6208e45d6e73fa86716d2e9a2ade5ab464cde655acd45c7e4d1801ed2d3626bab3ba05e3b41f2a29450865a40132f002991412938cb5af6ff8e93350a7074e39115f85f2288f44fe229c775174be5b832bb4b149636604b0f883b68a39b442fc044b4272b74b034560ad3c934a607eb017c1bb51509638366b61b3c029eaa03f3d1c3b66ce9d7358241f6e20ef73c842c6d48402df04c80b7916619f713d36121ec3c2cfcc8d862b03e2d6a80d0159cf138369c7a5bb811c26c8ce599ea5824eb97d92016897bd8ec3a912c3c0528e41a0038b90f8cd318ab4c16059f461803ee8419dfc0f2e3d54dbe4278dc449ee668fe6de2e0f9af103ce06baa5efe99d99c8fd65e23baa29b40857c890438a07bd5410c8f490482313a557b2a1b01f2eae6526043f933d3e5b606511ef29b85296423d082c1bf5f50e4f3d7b7129088a4392281a578fefb9e4f6d19089d59e27a3af9e38309d0f592f778bd7fe3474c69dded71857d1a1006f86e689dced68bd76988a56e2205e43ca6e0afa9d14289bd3289fe2f8135e93cedb9f99734ecd518bde28628a4a7933514ae6878154daf6ddf2b3865ad906b33a85b8f58f00cb3823b636f2ab3a5bcc92ddbf91897c0f767adca6337d28a6063ba239ee6
```

Comment:
Where is the WASM blob (genesis @ 0x3a636f6465)?

I also think we have to use the raw format to specify genesis contents.